docs/source/en/api/models/vq.mdx

```diff
@@ -0,0 +1,15 @@
+# VQModel
+
+The VQ-VAE model was introduced in [Neural Discrete Representation Learning](https://huggingface.co/papers/1711.00937) by Aaron van den Oord, Oriol Vinyals and Koray Kavukcuoglu. The model is used in 🤗 Diffusers to decode latent representations into images.
```

Comment:
Might be nice to include that as opposed to the AutoencoderKL model, VQModel works on a quantized latent space.